toolbar-ui-angular/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "toolbar-ui",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build --base-href ./",
+    "test": "ng test",
+    "test:watch": "ng test --code-coverage",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "author": "Tick42",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@angular/animations": "~7.2.0",
+    "@angular/common": "~7.2.0",
+    "@angular/compiler": "~7.2.0",
+    "@angular/core": "~7.2.0",
+    "@angular/forms": "~7.2.0",
+    "@angular/platform-browser": "~7.2.0",
+    "@angular/platform-browser-dynamic": "~7.2.0",
+    "@angular/router": "~7.2.0",
+    "@ng-bootstrap/ng-bootstrap": "^4.0.3",
+    "bootstrap": "^4.3.0",
+    "core-js": "^2.5.4",
+    "file-saver": "^2.0.1",
+    "ngx-bootstrap": "^3.2.0",
+    "ngx-clipboard": "^11.1.9",
+    "ngx-electron": "^2.1.1",
+    "rxjs": "~6.3.3",
+    "tslib": "^1.9.0",
+    "zone.js": "~0.8.26"
+  },
+  "devDependencies": {
+    "@angular-devkit/build-angular": "~0.13.0",
+    "@angular/cli": "~7.3.1",
+    "@angular/compiler-cli": "~7.2.0",
+    "@angular/language-service": "~7.2.0",
+    "@types/electron": "^1.6.10",
+    "@types/jasmine": "~2.8.8",
+    "@types/jasminewd2": "~2.0.3",
+    "@types/node": "~8.9.4",
+    "codelyzer": "~4.5.0",
+    "electron": "^4.0.4",
+    "electron-packager": "^13.0.1",
+    "glue-interop-api-impl": "^1.1.4",
```

Comment:
This dependency was not found.

Reply:
When contributed, I've validated the build of the production bundle, which works, although this missing dependency gets flagged as optional but missing:

```
npm WARN @plexus-interop/transport-common@0.4.0 requires a peer of rxjs@^5.4.2 but none is installed. You must install peer dependencies yourself.
npm WARN @plexus-interop/websocket-transport@0.4.0 requires a peer of rxjs@^5.4.2 but none is installed. You must install peer dependencies yourself.
npm WARN finos-fdc3-appd-toolbar@1.0.0 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: tick42-glue-core@4.3.4 (node_modules/tick42-glue-core):
npm WARN 404 SKIPPING OPTIONAL DEPENDENCY: Not Found - GET https://registry.npmjs.org/tick42-glue-core - Not found
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: glue-interop-api-impl@1.1.4 (node_modules/glue-interop-api-impl):
npm WARN 404 SKIPPING OPTIONAL DEPENDENCY: Not Found - GET https://registry.npmjs.org/glue-interop-api-impl - Not found
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: com-glue42-finos-fdc3-api-impl@1.0.3 (node_modules/com-glue42-finos-fdc3-api-impl):
npm WARN 404 SKIPPING OPTIONAL DEPENDENCY: Not Found - GET https://registry.npmjs.org/com-glue42-finos-fdc3-api-impl - Not found
```

These the steps to reproduce it:
```
npm install -g node-license-validator
git clone git@github.com:Tick42/finos-fdc3-appd-toolbar.git
cd finos-fdc3-appd-toolbar/
npm install --production
cd toolbar-ui-angular
npm install --production
```

More info on [this JIRA comment](https://finosfoundation.atlassian.net/browse/CONTRIB-43?focusedCommentId=12485&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-12485); hope this helps.

Reply:
@ikostova-Tick42 @lspiro-Tick42 Any background or context that the FDC3 PMC should consider re the change from "glue-fdc3-access-impl": "^1.0.11" to glue-interop-api-impl": "^1.1.4" does in the 2nd set of commits on this PR from 4/24?